apps/fluent-tester/src/FluentTester/TestComponents/Icon/IconTest.tsx

```diff
@@ -11,34 +11,76 @@ const testTtf = require('./assets/Font Awesome 5 Free-Solid-900.otf');
 import TestSvg from './assets/test.svg';
 
 const icons: React.FunctionComponent<{}> = () => {
-  const rasterProps: RasterImageIconProps = { src: testImage };
+  const fontCustomFontProps: FontIconProps = {
+    fontFamily: 'Font Awesome 5 Free',
```

Comment:
Where does this fontFamily come from? Never heard of that one

Reply:
It's described here: https://www.w3schools.com/icons/fontawesome5_intro.asp